packages/runtime/test/src/interpreter/inner-transaction.ts

```diff
@@ -1409,6 +1410,25 @@ describe("Inner Transactions", function () {
 			});
 		});
 
+	});
+
+	describe("Teal v7 update", function () {
+		this.beforeEach(() => {
+			setUpInterpreter(1, ALGORAND_ACCOUNT_MIN_BALANCE);
+		});
+		describe("Inner transaction can run in v7", () => {
```

Comment:
Why do we have to describes and why do we setup the interpreter twice?

Reply:
for this one, I want to set the interpreter teal version into 1
but I will. change a bit

Reply:
also we describe here since we also solve an issue:
https://github.com/scale-it/algo-builder/issues/759